src/coveoua/simpleanalytics.ts

```diff
@@ -137,6 +138,10 @@ export class CoveoUA {
         this.plugins = new Plugins();
         this.params = {};
     }
+
+    version(): string {
+        return `${libVersion}`;
```

Comment:
Is there a reason to put this inside a template literal? I'd simply expect
```suggestion
        return libVersion;
```

Reply:
Leftover from a previous change :) I'll fix it.